CMakeLists.txt

```
@@ -121,12 +121,20 @@ if(_use_onemkl_interfaces)
     else()
         FetchContent_Declare(
                 onemkl_interfaces_library
-                GIT_REPOSITORY https://github.com/oneapi-src/oneMKL.git
+                GIT_REPOSITORY https://github.com/uxlfoundation/oneMath.git
                 GIT_TAG 8f4312ef966420b9b8b4b82b9d5c22e2c91a1fe7  # v0.6
         )
     endif()
 
     FetchContent_MakeAvailable(onemkl_interfaces_library)
+    if(TARGET onemath)
+        set(USED_MATH_LIBRARY "onemath" CACHE INTERNAL "Selected OneAPI math library")
+    elseif(TARGET onemkl)
+        set(USED_MATH_LIBRARY "onemkl" CACHE INTERNAL "Selected OneAPI math library")
+    else()
+        message(STATUS "Neither 'oneMath' nor 'oneMKL' found!")
```

Should it be error then?
```suggestion
        message(FATAL_ERROR "Neither 'oneMath' nor 'oneMKL' found!")
```